classes/collector/base.php

```diff
@@ -56,4 +56,13 @@ public function record_metrics(array $metrics) {
     public function is_ready(): bool {
         return true;
     }
+
+    /**
+     * Abilitity for a collector to save old data.
+     *
+     * @return bool
+     */
+    public function supports_saved_metric_record(): bool {
```

Comment:
The method below is called is_backfillable, I think we should be consistent with the names in code and comments, eg something like:

is_backfillable()
supports_backfillable_metrics()